src/pyfaf/bugtrackers/mantisbt.py

```diff
@@ -0,0 +1,246 @@
+# Copyright (C) 2014  ABRT Team
+# Copyright (C) 2014  Red Hat, Inc.
+#
+# This file is part of faf.
+#
+# faf is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# faf is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with faf.  If not, see <http://www.gnu.org/licenses/>.
+
+from __future__ import absolute_import
+#from __future__ import unicode_literals
+
+from suds.client import Client
+
+from pyfaf import queries
+from pyfaf.utils.decorators import retry
+
+from pyfaf.storage.mantisbt import MantisBug
+
+from pyfaf.bugtrackers import BugTracker
+
+__all__ = ["Mantis"]
+
+
+BUG_STATE_MAP = {
+    "new": "NEW",
+    "feedback": "NEW",
+    "acknowledged": "NEW",
+    "confirmed": "NEW",
+    "assigned": "ASSIGNED",
+    "resolved": "CLOSED",
+    "closed": "CLOSED"
+}
+
+BUG_RESOLUTION_MAP = {
+    "open": None,
+    "fixed": "UPSTREAM",
+    "reopened": None,
+    "unable to reproduce": "WORKSFORME",
+    "not fixable": "CANTFIX",
+    "duplicate": "DUPLICATE",
+    "not a bug": "NOTABUG",
+    "suspended": "INSUFFICIENT_DATA",
+    "won't fix": "WONTFIX"
+}
```

Comment:
Yeah, I've also tried to find these but did only find that one in template which is CLOSED that should be uniform across bugtrackers. The one in filter is not really important as it only maps resolutions to bootstrap label names.

So I'm not really convinced that it's worth mapping stuff to what's used in bugzilla as we loose some potentially valuable information like `feedback` status. Also how do you map back if you want to alter these bugs from the server?